app/views/mentor_application_mailer/confirm_application.html.slim

```diff
@@ -1,7 +1,7 @@
 p
   = "Dear #{@mentor.first_name},"
 p
-  |Thank you for applying to "Learn IT, Girl" second edition! It is great to know you want
+  |Thank you for applying to "Learn IT, Girl" second edition! It is great to know you want 
```

Comment:
Hi @anjali-sharma :)
Instead of adding empty space could you change `=>` to `=< ` ?

Thanks :)